simtools/ray_tracing/ray_tracing.py

```diff
@@ -108,38 +101,39 @@ def __repr__(self):
 
     def _initialize_config(
```

Comment:
A little bit hard to follow the code here: the config is not initialized here but actually now later directly in SimulatorRayTracing. Maybe renaming/deleting the function and having functions that each set self.mirror_number and self.source_distance ?

Reply:
Renamed this function to `_initialize_mirror_configuration` to make it clearer. Note the comment on source_distance below.